src/ashtech.cpp

```diff
@@ -942,7 +942,7 @@ int GPSDriverAshtech::waitForReply(NMEACommand command, const unsigned timeout)
 	return _command_state == NMEACommandState::received ? 0 : -1;
 }
 
-int GPSDriverAshtech::configure(unsigned &baudrate, OutputMode output_mode)
+int GPSDriverAshtech::configure(unsigned &baudrate, OutputMode output_mode, GNSSSystemsMask gnssSystems)
```

Comment:
As this is an interface change (and therefore requires all users of the drivers to update, so PX4 + QGC, and unknown ones), I'd rather create a separate struct to simplify future additions:
```suggestion
int GPSDriverAshtech::configure(unsigned &baudrate, const GPSConfig& config)
```
```
struct GPSConfig {
  OutputMode output_mode = xy;
  GNSSSystemsMask gnss_systems = xy;
};

Reply:
Initially I thought of using a default value for the parameter, so it doesn't break things. But then I left it out because I haven't had in mind, that other projects than PX4 will use this repo.
But you suggestion is actually the cleaner and more future proof approach, since we breaks things with this anyways.